docs/blog/20240115-Daeraxa-v1.113.0.md

```diff
@@ -0,0 +1,59 @@
+
+We have a rather significant update to our `symbols-view` package. This package is used to display symbols within Pulsar (e.g. function definitions) and allows you to navigate your code via those symbols. This package has had a rather major overhaul and now follows the standard "provider/consumer" model as many other Pulsar packages do. In particular, this allows for packages to provide symbols to `symbols-view`, such as Tree-sitter grammars, via the new `symbol-provider-tree-sitter` package. The upshot of this, combined with the new Tree-sitter implementation, is that there will be a much richer and more accurate display of symbols in your project. More information can be found in the [pull request](https://github.com/pulsar-edit/pulsar/pull/829) for this change.
+
+A new Tree-sitter PHP grammar has been added to Pulsar as part of ongoing Tree-sitter grammar upgrades and improvements, which includes some contributions from community member [@claytonrcarter](https://github.com/claytonrcarter) who had a parser for PHPDoc allowing us to highlight documentation comments in PHP in a similar vein to JSDoc injection JS/TS files.
+
+On to some bug fixes. We had an issue reported about the `github` package that showed a problem with the rendering of the diff view display. This seems to have been due to a code path in our TextEditor component that would sometimes try to perform measurements on things that weren't necessarily visible. The fix here, simple as it seems, was to defer those measurements until we could be sure the editor was visible.
+
+Next, we have a fix to the `find-and-replace` package caused by an odd interaction between the package and the `Preserve Case During Replace` option, causing an error to appear when using empty strings as input (as an empty string cannot be capitalized).
+
+And last, we have a change to the default behavior of rendering emojis in Markdown as part of the new `UI` API. This issue was discovered due to an unwelcome (but admittedly rather apt) 😡 in the middle of an error message popup. While [the error](https://github.com/pulsar-edit/pulsar/pull/850) was rather hilarious, we did decide that it needed to be banished by default for the sake of one of our team members' sanity (and to reduce unnecessary obfuscation of errors, of course).
```

Comment:
```suggestion
And last, we have a change to the default behavior of rendering emojis in Markdown as part of the new `UI` API. This issue was discovered due to an unwelcome (but admittedly rather apt) 😡 emoji in the middle of an error message popup. While [the error](https://github.com/pulsar-edit/pulsar/pull/850) was rather hilarious, we did decide that it needed to be banished by default for the sake of one of our team members' sanity (and to reduce unnecessary obfuscation of errors, of course).
```

Clarify that the emoji itself is a subject in the sentence, that it's not flavoring the adjective "apt" ?

Reply:
Fine with me but should we also update the release post if it hasn't been done?